protos/sigstore_bundle.proto

```diff
@@ -48,7 +48,14 @@ message TimestampVerificationData {
 }
 
 // VerificationMaterial captures details on the materials used to verify
-// signatures.
+// signatures. This message may be embedded in a DSSE envelope as a signature
+// extension. Specifically, the `ext` field of the extension will expect this
+// message when the signature extension is for Sigstore. This is identified by
+// the `kind` field in the extension, which must be set to
+// application/vnd.dev.sigstore.verificationmaterial;version=0.1 for Sigstore.
+// When used as a DSSE extension, if the `public_key` field is used to indicate
+// the key identifier, it MUST match the `keyid` field of the signature the
+// extension is attached to.
```

Comment:
Just to make sure I'm not missing anything: this is the PR's only change, right? I don't see any protobuf definition changes 🙂

Reply:
Yes, the doc is the only change. This PR originally proposed a new message that was a subset of verification materials, but the feedback was to just use verification materials directly. The doc change unblocks https://github.com/secure-systems-lab/dsse/pull/61 which introduces the sigstore DSSE extension.

> the doc change doesn't seem actionable to a Sigstore client implementer

WRT a sigstore client, it's a question of whether it will support a DSSE envelope that uses the signature extension as an input to signature verification. See: https://github.com/sigstore/sig-clients/issues/9 and the follow up https://github.com/sigstore/sig-clients/issues/9#issuecomment-1665979019.